### PR TITLE
Contribution guide

### PR DESCRIPTION
**Pull request by beam2d**
I wrote the contribution guide. This document includes our development process, issues and PRs policy, and coding/testing guidelines.

### CODE REVIEW

File: docs/source/contribution.rst [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+------------------
+
+Testing is one of the most important part of your code.
+You must test your code by unit tests following our testing guidelines.
+
+We are using ``nose`` package to run unit tests.
+You can run unit tests simply by running ``nosetests`` command under the repository root.
+It requires CUDA by default.
+In order to run unit tests that do not require CUDA, pass ``--attr='!gpu'`` option to the nosetests command.
+
```
Comment (unnonouno) [inline docs/source/contribution.rst]: How about writing an example of a command like this?
```
nosetests tests/path/to/your/test.py -a '!gpu'
```

**Comment by unnonouno** [comment]
It looks good. Does it need proofreading by natives?

**Comment by beam2d** [comment]
Thank you for the review. I added a command line example of CPU-only test (and simultaneously added a whole-test example).

**Comment by delta2323** [comment]
cf. #83  (to make this PR visible from original issue).

**Comment by beam2d** [comment]
@delta2323 Thank you for adding a reference.
@unnonouno This PR affects all on-going PRs, so I think we should merge this now. After the merge, I will ask someone for proofreading.

**Comment by unnonouno** [comment]
OK. I'll merge.

**Comment by beam2d** [comment]
Thanks!

**Pull request merged**